newdle_style.css

```diff
@@ -1,184 +1,163 @@
-    background: rgb(125, 159, 156);
-    z-index: 3;
-    visibility: hidden;
+  position: absolute;
+  height: 93vh;
+  width: 87vw;
+  top: 4vh;
+  left: 6vw;
+  background: rgb(125, 159, 156);
+  z-index: 1;
+  visibility: hidden;
 }
 
 #exit_button{
-    max-width: 5vw;
-    position: absolute;
-    top: 2vh;
-    left: 77vw;
-    
+  max-width: 5vw;
+  position: absolute;
+  top: 2vh;
+  left: 77vw; 
 }
 
 #instructions_headline{
-    font-weight: bold;
-    font-size: 2rem;
-    position: absolute;
-    text-align: center;
-    left: 21vw;
+  font-weight: bold;
+  font-size: 2rem;
+  position: absolute;
+  text-align: center;
+  left: 21vw;
 }
 
 #instuctions {
-  position: relative;
-  top: 14vh;
-  padding: 6vw;
-  font-size: 1.5rem;
+position: relative;
+top: 14vh;
+padding: 6vw;
+font-size: 1.5rem;
 }
 
-
 #example{
-  position: relative;
-  left: 25vw;
-  top: 10vh;
-  font-size: 1rem;
-}
-
-#first_background{
-    position: absolute;
-    top: 0px;
-    left: 0;
-    bottom: 0px;
-    right: 0px;
-    background:slategray;
-    z-index: -3;
-}
-
-#second_background{
-    position: absolute;
-    background:black;
-    top: 2vh;
-    height: 97vh;
-    width: 94vw;
-    left: 2vw;
-    z-index: -2;
+position: relative;
+left: 25vw;
+top: 10vh;
+font-size: 1rem;
 }
 
 #foreground{
-    position: relative;
-    height: 93vh;
-    width: 87vw;
-    top: 4vh;
-    left: 6vw;
-    background:rgb(94, 82, 194);
-    z-index: 2;
-    text-align: center;
-
+  position: relative;
+  height: 93vh;
+  width: 87vw;
+  top: 4vh;
+  left: 6vw;
+  background:rgb(94, 82, 194);
+  z-index: 2;
+  text-align: center;
+  border-color: black;
+  border-width: 2vw;
+  border-style: solid;
 }
 
 /* Information in foreground */
 
 #instructions_button{
-    max-width: 3vh;
-    position: relative;
-    left: 36vw;
-    top: 1.5vh;
+  max-width: 5vh;
+  position: relative;
+  left: 36vw;
+  top: 1.5vh;
 }
 
 #heading{
-    letter-spacing: 3vw;
-    font-size: 13vw;
-    text-align:inherit;
-    position: relative;
-    left: 0vw;
-    z-index: 0;
+  letter-spacing: 3vw;
+  font-size: 13vw;
+  text-align:inherit;
+  position: relative;
+  left: 0vw;
+  z-index: 0;
 }
 
 #congratulations_text{
-    font-size: 5vw;
-    font-weight: bold;
-    z-index: 0;
+  font-size: 5vw;
+  font-weight: bold;
+  z-index: 0;
 }
 #presented_headline{
-    line-height: 8vh;
-    font-size: 7vw;
-    padding: 4vw;
-    text-align: justify;
+  line-height: 8vh;
+  font-size: 7vw;
+  padding: 4vw;
+  text-align: center;
 }
 #guess_box{
-    position: relative;
-    font-size: 60%;
-    top: 1vh;
-    width: 36vw;
-    height: 4vh;
-    z-index: 0;
+  position: relative;
+  font-size: 60%;
+  top: 1vh;
+  width: 36vw;
+  height: 4vh;
+  z-index: 0;
 }
 #submit_button{
-    font-size: 6vw;
-      align-items: center;
+  font-size: 6vw;
+  align-items: center;
   background-color: #fff;
   border: 2px solid #000;
-  box-sizing: border-box;
+  /*! box-sizing: border-box; */
```

Comment:
I'd recommend not leaving in commented out code.
If you want to look back at previously used properties then you can use the git history